src/database/queries/userExtension.js

```diff
@@ -80,6 +80,7 @@ module.exports = class MenteeExtensionQueries {
 					external_mentor_visibility: null,
 					deleted_at: Date.now(),
 					org_id,
+					status: null,
```

Comment:
why status is null? @sumanvpacewisdom

Reply:
@ankitpws it should be string , i missed it will change